eng/common/TestResources/SubConfig-Helpers.ps1

```diff
@@ -5,7 +5,7 @@ function BuildServiceDirectoryPrefix([string]$serviceName) {
 # If the ServiceDirectory has multiple segments use the last directory name
 # e.g. D:\foo\bar -> bar or foo/bar -> bar
 function GetServiceLeafDirectoryName([string]$serviceDirectory) {
-    return Split-Path -Leaf $serviceDirectory
+    return $serviceDirectory ? (Split-Path -Leaf $serviceDirectory) : ""
```

Comment:
Is this valid PS syntax? I didn't think PS supported this operator.

Reply:
wow this is a new PS 7.0 feature that I didn't realize they added https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_if?view=powershell-7.2#using-the-ternary-operator-syntax

Reply:
Yep and they also supprt null coalescing `??` !

Reply:
And null condition operators as well https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_operators?view=powershell-7.2#null-conditional-operators--and- These are all new to me so I will have to remember to try them out.

Reply:
Unfortunately they aren't supported for strict mode (yet, hopefully).